po/vi.po

```diff
@@ -6254,138 +6254,168 @@ msgid ""
 
 #: src/std-types/vec.md
 msgid ""
 "`vec![...]` is a canonical macro to use instead of `Vec::new()` and it "
 "supports adding initial elements to the vector."
 msgstr ""
+"Ta có thể sử dụng macro `vec![...]` thay vì gọi hàm `Vec::new()` để khởi tạo "
+"một `Vec`. `vec![...]` còn có thể được dùng để khởi tạo vector với một số "
+"phần tử cho trước."
 
 #: src/std-types/vec.md
 msgid ""
 "To index the vector you use `[` `]`, but they will panic if out of bounds. "
 "Alternatively, using `get` will return an `Option`. The `pop` function will "
 "remove the last element."
 msgstr ""
+"Để truy cập phần tử của một vector ta có thể sử dụng `[` `]`, nhưng nếu phần "
+"tử được truy cập cập nằm ngoài phạm vi của vector, chương trình sẽ panic. "
+"Trong trường hợp này, ta có thể sử dụng hàm `get` để trả về một `Option`. "
+"Hàm `pop` sẽ loại bỏ phần tử cuối cùng của vector."
 
 #: src/std-types/vec.md
 msgid ""
 "Slices are covered on day 3. For now, students only need to know that a "
 "value of type `Vec` gives access to all of the documented slice methods, too."
 msgstr ""
+"Chúng ta sẽ đi sâu vào chi tiết về slice trong ngày 3. Bây giờ, học viên chỉ "
+"cần biết rằng một biến kiểu `Vec` cũng có thể sử dụng tất cả các hàm của "
+"slice."
 
 #: src/std-types/hashmap.md
 msgid "Standard hash map with protection against HashDoS attacks:"
 msgstr ""
+"Một bảng băm chuẩn với cơ chế bảo vệ chống lại các cuộc tấn công HashDoS:"
 
 #: src/std-types/hashmap.md
 msgid "\"Adventures of Huckleberry Finn\""
-msgstr ""
+msgstr "\"Những cuộc phiêu lưu của Huckleberry Finn\""
 
 #: src/std-types/hashmap.md
 msgid "\"Grimms' Fairy Tales\""
-msgstr ""
+msgstr "\"Truyện cổ tích của Grimms\""
 
 #: src/std-types/hashmap.md
 msgid "\"Pride and Prejudice\""
-msgstr ""
+msgstr "\"Kiêu hãnh và định kiến\""
 
 #: src/std-types/hashmap.md
 msgid "\"Les Misérables\""
-msgstr ""
+msgstr "\"Những người khốn khổ\""
 
 #: src/std-types/hashmap.md
 msgid "\"We know about {} books, but not Les Misérables.\""
-msgstr ""
+msgstr "\"Chúng ta biết {} cuốn sách, nhưng không biết về Les Misérables.\""
 
 #: src/std-types/hashmap.md
 msgid "\"Alice's Adventure in Wonderland\""
-msgstr ""
+msgstr "\"Alice lạc vào xứ sở thần tiên\""
 
 #: src/std-types/hashmap.md
 msgid "\"{book}: {count} pages\""
-msgstr ""
+msgstr "\"{book}: {count} trang\""
 
 #: src/std-types/hashmap.md
 msgid "\"{book} is unknown.\""
-msgstr ""
+msgstr "\"{book} không tồn tại.\""
 
 #: src/std-types/hashmap.md
 msgid "// Use the .entry() method to insert a value if nothing is found.\n"
 msgstr ""
+"// Sử dụng phương thức .entry() để chèn một giá trị nếu giá trị đó chưa tồn "
+"tại trong hashmap.\n"
 
 #: src/std-types/hashmap.md
 msgid "\"{page_counts:#?}\""
-msgstr ""
+msgstr "\"{page_counts:#?}\""
 
 #: src/std-types/hashmap.md
 msgid ""
 "`HashMap` is not defined in the prelude and needs to be brought into scope."
 msgstr ""
+"`HashMap` không được định nghĩa trong prelude và cần phải được import vào "
```

Comment:
Suggest translating HashMap

Reply:
I don't think this is necessary anymore. Can you confirm this @qu-ngx?